nl.brixit.wiremapper.json

```diff
@@ -0,0 +1,59 @@
+{
+  "app-id": "nl.brixit.wiremapper",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.36",
+  "sdk": "org.gnome.Sdk",
+  "command": "wiremapper",
+  "finish-args": [
+    "--socket=wayland",
+    "--socket=fallback-x11",
+    "--share=ipc",
+    "--system-talk-name=org.bluez",
+    "--allow=bluetooth",
+    "--share=network"
+  ],
+  "modules": [
+    "python3-wiremapper.json",
+    {
+      "name": "wiremapper"
+    },
+    {
+      "name": "libhandy",
+      "buildsystem": "meson",
+      "config-opts": [
+        "-Dprofiling=false",
+        "-Dintrospection=enabled",
+        "-Dgtk_doc=false",
+        "-Dtests=false",
+        "-Dexamples=false",
+        "-Dvapi=false",
+        "-Dglade_catalog=disabled"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://source.puri.sm/Librem5/libhandy.git",
+          "tag": "v0.0.13",
+          "commit": "7a193d7692c9c76a1a94f17c4d30b585f77d177c"
+        }
+      ]
+    },
+    {
+      "name": "desktop-files",
+      "buildsystem": "simple",
+      "build-commands": [
```

Comment:
Any reason for not installing the wiremapper package here and using the source from pip? that's what most apps do and they generate the sources from a requirements.txt file or something equivalent for the app dependencies

Reply:
This should also allow you to use the manifest with GNOME Builder/Gitlab CI

Reply:
I like to keep my non-python-module stuff seperate from it since installing that stuff using pip is a bit of a hack.

Reply:
Well, in such case, just use a file source for the metainfo/desktop/icon to not clone the whole repository